index.html

```diff
@@ -43,6 +43,172 @@ <h3>LET'S CONNECT</h3>
+                    <h4>CANOPY</h4>
+                    <ul class="ul-list">
+                        <li>Back End Dev</li>
+                    </ul>
+                    <ul>
+                        <li>2015</li>
+                    </ul>
+                </div>
+                <div class="paragraph">
+                    <h4>
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </h4>
+                </div>
+                <div class="btn">
+                    <button type="button" class="btn-1">html</button>
+                    <button type="button" class="btn-2">css</button>
+                    <button type="button" class="btn-3">javaScript</button>
+                </div>
+                <div class="big-btn">
+                    <button type="button" class="btn-4">See Project</button>
+                </div>
+
+        </section>
+        <section class="subsection-two">
+            <img src="images/Snapshoot-Portfolio-2.png" alt="">
+               <div>
+                <h3>
+                    Multi-Post Stories
+                </h3>
+            </div>
+            <div class="list-items">
+                <h4>CANOPY</h4>
+                <ul class="ul-list">
+                    <li>Back End Dev</li>
+                </ul>
+                <ul class="ul-list">
+                    <li>2015</li>
+                </ul>
+            </div>
+            <div class="paragraph">
+                <h4>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </h4>
+            </div>
+            <div class="btn">
+                <button type="button" class="btn-1">html</button>
+                <button type="button" class="btn-2">css</button>
+                <button type="button" class="btn-3">javaScript</button>
+            </div>
+            <div class="big-btn">
+                <button type="button" class="btn-4">See Project</button>
+            </div>
+        </section>
+        <section class="subsection-three">
+            <img src="images/Snapshoot-Portfolio-3.png" alt="">
+            <div>
+             <h3>
+                 Tonic
+             </h3>
+         </div>
+         <div class="list-items">
+             <h4>CANOPY</h4>
+             <ul class="ul-list">
+                 <li>Back End Dev</li>
+             </ul>
+             <ul class="ul-list">
+                 <li>2015</li>
+             </ul>
+         </div>
+         <div class="paragraph">
+             <h4>
+                 A daily selection of privately personalized reads; no accounts or sign-ups required.
+             </h4>
+         </div>
+         <div class="btn">
+             <button type="button" class="btn-1">html</button>
+             <button type="button" class="btn-2">css</button>
+             <button type="button" class="btn-3">javaScript</button>
+         </div>
```

Comment:
- [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list ul and each of them a li. kindly change this.